bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/Handler.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.bookkeeper.client.api;
+
+import java.util.concurrent.CompletableFuture;
+import org.apache.bookkeeper.client.BKException;
+
+/**
+ * Handle a Ledger
+ */
+public interface Handler extends AutoCloseable {
```

Comment:
Handler or Handle?

Reply:
The original proposal is "Handler", I need to fix up javadocs

Reply:
"Handler" is a bit strange in this context. It hints towards these being callbacks of sorts, as in a "Chain-of-responsibility" pattern.
Handle makes more sense, because you're getting a reference to a remote object (i.e. the ledger) which you can then perform operations on.

Reply:
@ivankelly this make sense to me, in fact the original class is LedgerHandle, it is like a file descriptor.
done